bin/pd/Makefile

```diff
@@ -278,11 +279,20 @@ upload-consultations: $(workdir)/filtered/consultations.csv \
 		$(workdir)/filtered/current_consultations_open.csv
 	@echo Publishing Consultations to registry and portal...
 	cp $(workdir)/consultations.csv $(registry_static)
+	@$(ckanapi) action resource_patch -c $(registry_ini) \
+	id=897d3008-b258-4a68-8c02-3e3c099a42c8 last_modified=$(isodate) \
+	> /dev/null
```

Comment:
this one updates a resource date that only a few users on the registry can see. Is that what you intended?

Reply:
Yep - I was specifically looking for this resource - https://registry.open.canada.ca/en/dataset/60f83adb-30e5-4819-8300-1bd502576507/resource/897d3008-b258-4a68-8c02-3e3c099a42c8 

We've had issues troubleshooting the consultations when users report their data isn't published - checking the last_modified date on this file should be a useful datapoint for troubleshooting